src/components/reusable/tag/tag.scss

```diff
@@ -229,117 +229,116 @@
       }
     }
   }
-  /** NOTE: Here onwards color has HEX value as there is no color pallete for Data Viz yet **/
   /* cat01 color */
   &-cat01 {
-    background-color: #ace1f8;
+    background-color: var(--kd-color-data-sea-10);
```

Comment:
Use a named token instead of a palette token where at all possible.

Reply:
Updated